utils/scripts/ocm/ocm.py

```diff
@@ -1179,6 +1179,7 @@ def change_cluster_channel_group(self):
         ret = execute_command(run_change_channel_cmd)
         if ret is None:
             log.info("Failed to update the channel to {}".format(self.cluster_name))
+            sys.exit(1)
```

Comment:
I think it better if you check for the exiting channel on the cluster before making and modification. If channel is "Candidat" the do not modify it .

Reply:
sure  that can be done in different PR this pr is just for sys.exit(1)